app/views/supervisor/dashboard/index.html.slim

```diff
@@ -1,28 +1,29 @@
 h2 Supervisor Dashboard
 p Alpha Version
 
-div.clearfix
+div.row
   - supervised_teams.each do |t|
-    div.dashboard-box
-      section.team-title
-        h3 = link_to "Team " + t.name, t
-      section.dashboard
-        h4 Checks
-        - if t.comments.any?
-          p = "You last checked on them #{ time_ago_in_words(t.comments.last.created_at) } ago."
-        - else
-          p You didn't check on them yet.
-        = simple_form_for Comment.new(team_id: t.id), url: supervisor_comments_path do |f|
-          == render 'supervisor/comments/form', f: f
+    div.dashboard-box.col-sm-6 class=("col-lg-4" if supervised_teams.count > 2)
+      div.background-color class=("min-height" if supervised_teams.count > 2)
+        section.team-title
+          h3 = link_to "Team " + t.name, t
+        section.dashboard
+          h4 Checks
+          - if t.comments.any?
+            p = "You last checked on them #{ time_ago_in_words(t.comments.last.created_at) } ago."
+          - else
+            p You didn't check on them yet.
+          = simple_form_for Comment.new(team_id: t.id), url: supervisor_comments_path do |f|
+            == render 'supervisor/comments/form', f: f
           - if t.comments.any?
             p = render t.comments.recent
             p = link_to 'Archive', supervisor_comments_path(team_id: t.id)
-      section.dashboard
-        h4 Activity
-        - if t.activities.any?
-          p = "Their latest update is from #{ time_ago_in_words(t.last_activity.created_at) } ago."
-          h5 = t.last_activity.title
-          p = render_markdown(t.last_activity.content).html_safe
-        - else
-          p Your team hasn't written any status updates yet.
+        section.dashboard
+          h4 Activity
+          - if t.activities.any?
+            p = "Their latest update is from #{ time_ago_in_words(t.last_activity.created_at) } ago."
+            h5 = t.last_activity.title
+            p = truncate((t.last_activity.content).html_safe, length: 220) { link_to " Show all", activities_path(team_id: t.id) }
```

Comment:
the `link_to` here doesn't work. 
If you want to have it all on the same line, you can put it in a block:
```
p do
= truncate(...
= link_to ..
````
Another possibility is to add it to the next line, that also works :)